product_template_multi_link/wizards/product_template_linker.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ProductTemplateLinker(models.TransientModel):
+    """
+    Wizard used to link product template together in one shot
+    """
+    _name = "product.template.linker"
+    _description = "Product template linker wizard"
+
+    operation_type = fields.Selection(
+        selection=[
+            ('unlink', 'Remove existing links'),
+            ('link', 'Link these products'),
+        ],
+        string="Operation",
+        required=True,
+        help="Remove existing links: will remove every existing link "
+             "on each selected products;\n"
+             "Link these products: will link all selected "
+             "products together.",
+    )
+    product_ids = fields.Many2many(
+        comodel_name="product.template",
+        string="Products",
+    )
+    link_type = fields.Selection(
+        string='Link Type',
+        selection=lambda self: self.env[
+            'product.template.link']._selection_link_type(),
+        default='cross_sell',
+        help="* Cross-Sell: suggest your customer to  purchase an "
+             "additional product;\n"
+             "* Up-Sell: suggest your customer to purchase a "
+             "higher-end product, an upgrade, etc.")
+
+    @api.model
+    def default_get(self, fields_list):
+        """
+        Inherit default_get to auto-fill product_ids with current context
+        :param fields_list: list of str
+        :return: dict
+        """
+        result = super(ProductTemplateLinker, self).default_get(fields_list)
+        ctx = self.env.context
+        active_ids = ctx.get('active_ids', ctx.get('active_ids', []))
```

Comment:
:thinking:

Reply:
Haha copy/paste.. It should be `active_id` in the second place :fearful: